exporter/awsemfexporter/grouped_metric.go

```diff
@@ -91,6 +91,15 @@ func addToGroupedMetric(
 				// them together into one EMF log event, so don't set batchIndex when it's a summary metric
 				metadata.batchIndex = i
 			}
+
+			// Handle metric types with container insights
+			if metadata.receiver == containerInsightsReceiver {
+				// For container insights, treat gauge metrics as sum
```

Comment:
nit: Can the comment explain why we're doing this?
```suggestion
				// For container insights, treat gauge metrics as sum to keep metrics in the same EMF record
```